misc/changelog-generator/instructions.md

```diff
@@ -0,0 +1,30 @@
+
+### Creating Pull Requests for Inclusion in NEWS.md
+
+If a PR is significant enough to warrant a mention in the next release notes update,
+its name should begin with a prefix. 
+There are three options depending on the PR's
+purpose.
+
+  * `breaking-change: ` if the change will make the next tag non-backward-compatible
+  * `feature: ` if the change is a major addition that maintains backward-compatibility
+  * `fix: ` if the change is a bug fix, security patch, or other improvement
+
+If the PR does not begin with one of these prefixes, it WILL NOT be included in
+the release notes, so make sure to name important PRs accordingly. 
+
+### Generating Notes and Creating a New Tag
+
+To update release notes for and commit a new git tag:
+
+1. Navigate to this directory in while on master branch
+2. Run `node changelog_generator.js` on the command line
+3. Confirm version was updated in package.json and notes were added to NEWS.md
+4. Push to **origin/master** (this can only be done by owners/admins).
+
+The generator follows semantic versioning, so:
+
+  * If there have been breaking changes since the last tag, it will increment the 1st (major) version digit by 1 and set the 2nd (minor) and 3rd (patch) to 0. (e.g. 0.6.8 &rightarrow; 1.0.0)
```

Comment:
It is likely that we will have some breaking changes while we are still in preview, but where we do not yet want to increment the major release to 1.0

Reply:
how do you want to handle those?

Reply:
is it possible to just not increment the major version?

Reply:
What if we do this: until we do a major release, we keep the major at 0, use minor to indicate breaking changes, and use the two digits of patch to indicate new features and patches.

e.g.

breaking change: 0.6.85 --> 0.7.00
new feature: 0.6.85 --> 0.6.90
patch: 0.6.85 --> 0.6.86

Then, once we do our release, we can do 1.0.0 and do traditional semantic versioning from then on.

@heemanshusuri for his thoughts